src/components/demos/write/Demo.js

```diff
@@ -83,7 +106,7 @@ export default function WriteDemo(props) {
       <Header {...props} showClose={true} />
       <Wrapper {...theme.wrapper}>
         {successfulLogin(apiOutput) ? (
-          <SecretDesktop {...theme.font} />
+          renderDesktopIcons(props.secretDesktop)
```

Comment:
Should be rendered using `jsx` notation as `renderDesktopIcons` is essentially a functional `React` component

Reply:
It has been updated.